src/components/document.tsx

```diff
@@ -210,17 +210,20 @@ export class DocumentComponent extends BaseComponent<IProps, {}> {
 
   private renderSupportIcons() {
     const supports = this.getSupportsWithIndices();
+    const anyActive = supports.some((support) => support.item.visible);
```

Comment:
FYI, argument parentheses are optional, so
```typescript
    const anyActive = supports.some(support => support.item.visible);
```
should work as well.

Reply:
I secretly like the parens! In case I need to add more params...but then again that almost never happens so maybe it's not worth it